services/src/test/java/org/apache/druid/cli/CliPeonTest.java

```diff
@@ -0,0 +1,73 @@
+package org.apache.druid.cli;
```

Comment:
Checkstyle will complain due to lack of license header. Suggestion: run checkstyle before each commit, it is fast:

```bash
mvn checkstyle:checkstyle -nsu
```

Reply:
oh nice, i didn't know there was a fast checkstyle.  I ran that before my last commit and things were good.